src/main/java/org/elasticsearch/index/analysis/TurkishStemmerTokenFilterFactory.java

```diff
@@ -12,8 +12,7 @@
 import org.elasticsearch.common.inject.assistedinject.Assisted;
```

Comment:
Probably reword the commit message with some more information about why this was required, changed and reverted. ( :scissors: sarcasm)